cmake/modules/kernel.cmake

```diff
@@ -142,7 +142,9 @@ enable_language(C CXX ASM)
 # Verify that the toolchain can compile a dummy file, if it is not we
 # won't be able to test for compatibility with certain C flags.
 zephyr_check_compiler_flag(C "" toolchain_is_ok)
-assert(toolchain_is_ok "The toolchain is unable to build a dummy C file. See CMakeError.log.")
+assert(toolchain_is_ok "The toolchain is unable to build a dummy C file.\
+ Move ${USER_CACHE_DIR}, re-run and look at \
+ ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/CMakeConfigureLog.yaml")
```

Comment:
and users running CMake 3.20 (which is minimum required) are having `CMakeFiles/CMakeError.log`.
So to such users referring to a file not there is just as confusing.

We need a slightly better fix 🙂  .

Reply:
seems to have moved location in 3.26: https://cmake.org/cmake/help/latest/manual/cmake-configure-log.7.html

and how nice of the CMake developers to inform us already:
> However, the location and name of the log file may change in future versions of CMake.

Reply:
OMG, so much over-engineering to just read an error message! Can we just reference the "cmake-configure-log corresponding to your CMake version" in the short term?

Reply:
i'm fine with a general description.
But if pointing to a specific file, then we should ensure such file exists.